scripts/pylib/twister/twisterlib.py

```diff
@@ -88,7 +88,6 @@
 logger = logging.getLogger('twister')
 logger.setLevel(logging.DEBUG)
 
-
```

Comment:
nit: just noticed this, I believe that the style guide is to have 2 blank lines before a class

Reply:
Done